_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Introducing Security Analytics"
+authors:
+  - christophermoore
+date:   2023-03-13 12:15:00 -0700
+categories:
+  - releases
+meta_keywords: security analytics
+meta_description: Provides and overview of Security Analytics and how it fits in with the OpenSearch Project
+---
+
+# Introducing Security Analytics
+
+In January of 2023, one of the largest wireless carriers in the US revealed that hackers had compromised a company database of customer information through unauthorized use of an API and had acquired the personal details of as many as 37 million individuals, including names, addresses, phone numbers, and account numbers. Once identified, the company was able to halt the malicious activity within a day. However, investigations indicated that the data breach had likely occurred at the end of November the previous year, giving the hackers over a month to extract the information.
+
+Although unfortunate, occurrences like these now appear frequently in the media and have become almost commonplace in the modern world. With an ever-increasing volume of business data shared online and stored in the cloud, malicious attempts to disrupt companies’ business interests are now simply part of the reality of doing business. And in light of these threats, it’s therefore essential that companies have tools in place to detect and identify the signatures of possible attacks in real time so they can act to intervene as quickly as possible. To address these needs, OpenSearch now includes a package of pre-built features and functionality that allow businesses to quickly and easily roll out a security solution for confronting a full range of cyber threats. Security Analytics is this solution for OpenSearch, and it installs automatically with any distribution.
+
+## The role of Security Analytics in OpenSearch
+
+OpenSearch already includes powerful features that allow administrators to track and analyze activity within the cluster. They provide users the ability to identify anomalous behavior in system data and set up alerts and notifications to signal users when these events occur. However, scaling these existing tools to serve the specific needs of various industries, handling the volume of data they possess, and then assembling a team of professionals to manage and interpret the results, pose a number of challenges.
+
+Security Analytics takes a different approach. It lets businesses use their existing OpenSearch cluster for detecting security threats, rather than requiring them to bear the cost of duplicating and storing their data for a separate off-the-shelf security solution. Security Analytics also leverages a schema of rules that it applies to log data to look for matches with abnormal events. These standardized rules address specific threats known throughout the industry and form a knowledge base of known attacks. Combined with a number of robust features, the rules also allow Security Analytics to be highly customizable.
```

Comment:
The significant cost savings isnt from using an existing OpenSearch cluster, but instead simply by using a single solution such as OpenSearch that provides a comprehensive suite of features and capabilities. In addition the lower cost and low latency performance at scale are important benefits that attract users to OpenSearch

Reply:
"Schema of rules" isnt accurate. Instead we could say "leverages widely used Sigma rules to find threat patterns in your security event logs"